enumerables.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
```

Comment:
There should be a condition after the line 4, that returns an enumerable, when no block is given